app/src/main/java/com/xinto/opencord/ui/screen/Chat.kt

```diff
@@ -326,6 +326,9 @@ private fun ChatScreenLoaded(
                                                 height = attachment.height
                                             )
                                         }
+                                        is DomainAttachment.Video -> {
+                                            WidgetAttachmentVideo(url = attachment.url)
```

Comment:
Is there no way to provide a static size so it doesn't resize when done buffering?

Reply:
It would be possible if I knew how to convert video dimensions to compose size

Reply:
Afaik discord still sends size params to video attachments, stable does it somehow